src/app/users/users.component.html

```diff
@@ -43,19 +43,29 @@
 </p-dialog>
 
 <p-dialog header="Confirm " [(visible)]="isOpenConfirmDeleteUserDialog">
-	Confirm you want to delete user {{userToDelete.email}}
+	Confirm you want to delete user {{userToDelete?.email}}
 	<br/><br/>
 	<button pButton pRipple type="button" label="Ok" class="p-button-raised p-button-text"
-		(click)="deleteUser(userToDelete.id)"></button>
+		(click)="deleteUser()"></button>
 </p-dialog>
 
 <p-dialog header="{{formType}} user " [(visible)]="isFormOpen">
 	<form [formGroup]="userForm" (ngSubmit)="submitForm()">
 		<input type="text" value="{{userToEdit?.firstName}}" placeholder="First name" id="firstName" formControlName="firstName" />
+		<div *ngIf="userForm.controls['firstName'].invalid && (userForm.controls['firstName'].dirty || userForm.controls['firstName'].touched)" class="alert" style="width: 200px; color: red;" >
```

Comment:
move styles to css and append to class
for ex:
.invalid {
  width: 200px; 
  color: red;
}
<div class=" alert invalid"...>...</div>